client/src/components/MainMenu.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Box from "./mainBoxComponent";
+import styles from './mainBox.module.css';
+import './button.css'
+
+export default function MainMenu() {
+  return (
+    <div className={styles.parent}>
```

Comment:
Change your mainBox.module.css to just MainMenu.css

Also replace your Box stuff with div (since it's just a div).

Then you can
```
<div className="parent"> 
    <div className="parent-box">
    </div>
</div>
```

^ Also notice that we use parent-box instead of parentBox since that's css convention 😎